bom/runtime/pom.xml

```diff
@@ -176,7 +176,7 @@
         <quarkus-spring-api.version>5.2.Final</quarkus-spring-api.version>
         <quarkus-spring-data-api.version>2.1.Final</quarkus-spring-data-api.version>
         <quarkus-spring-security-api.version>5.2.Final</quarkus-spring-security-api.version>
-        <quarkus-spring-boot-api.version>2.1.Final</quarkus-spring-boot-api.version>
+        <quarkus-spring-boot-api.version>2.2.Alpha1</quarkus-spring-boot-api.version>
```

Comment:
We need a Final version here, not an Alpha. This will be backported to 1.3.1.Final.

If 2.2.Final is not an option, use 2.2.SP1 or something like that.

/cc @geoand for verification.

Reply:
We agreed that the bump to 2.2 was not correct and we need to go back to 2.1.xxx

In any case @aureamunoz do keep in mind that in Quarkus master we can no longer have `Alpha` versions at any time.

Reply:
Ok, yes. Thanks. I will do the necessary in the API and open a PR, then update this one